scripts/processToFile.js

```diff
@@ -22,5 +24,29 @@ if(matches) {
 }
 
 var newFileName = mainProcessFile.replace('.template', '.js');
+
+// Look for <<param>>, see if there is a index attached
+// and replace that string with the corresponding argument.
+//   <<param>> will be replaced w/ process.argv[PARAM_START_INDEX]
+//   <<param2>> will be replaced w/ process.argv[PARAM_START_INDEX+2]
+regex = new RegExp('<<(param.*)>>', 'g');
+matches = mainProcess.match(regex);
+if(matches) {
+    var param_max = process.argv.length - PARAM_START_INDEX;
```

Comment:
@pmandler could you do a refactor of this change and then we can merge this PR?
I think my suggestion is clearer and easier to implement.